source/dialog/startup/startup_start2.sqf

```diff
@@ -617,7 +607,7 @@ publicVariable "opfor_ai_skill";
 publicVariable "enableChopperFastTravel";
 publicVariable "manually_chosen";
 
-	//	commandpointsblu1 = 9999999; 
+	//	commandpointsblu1 = 9999999;
 //hint format["Max radius: %1\nMin radius: %2\nZones number: %3\nCommand points: %4\nBLU AP: %5\nOPF AP: %6\nWeather type: %7\nBLU AI skill: %8\nOPF AI skill: %9",zones_max_radius,zones_min_radius,zones_number,commandpointsblu1,    opfor_ap,    blufor_ap,weather_type,    blufor_ai_skill,opfor_ai_skill];
 diag_log format["----- DUWS CHOSEN SETTINGS --------- Max radius: %1-------Min radius: %2-------Zones number: %3-------Command points: %4-------BLU AP: %5-------OPF AP: %6-------Weather type: %7-------BLU AI skill: %8-------OPF AI skill: %9",zones_max_radius,zones_min_radius,zones_number,commandpointsblu1,    opfor_ap,    blufor_ap,weather_type,    blufor_ai_skill,opfor_ai_skill];
-closeDialog 0;
\ No newline at end of file
+closeDialog 0;
```

Comment:
I really appreciate you fixing the white space on this file but I have bad news. This file goes away with pull request #59. You'll probably have a merge conflict once that gets merged in. The new CP rewards will go into the common sqf. I'm hoping that gets reviewed and merged today.